README.md

```diff
@@ -1,236 +1,93 @@
-│   │       └── signup.component.ts
-│   ├── assets
-│   │   ├── css
-│   │   ├── fonts
-│   │   ├── img=
-│   │   └── sass
-│   │       ├── paper-kit
-│   │       └── paper-kit.scss
-│   ├── environments
-│   ├── favicon.ico
-│   ├── index.html
-│   ├── main.ts
-│   ├── polyfills.ts
-│   ├── styles.scss
-│   ├── test.ts
-│   ├── tsconfig.app.json
-│   ├── tsconfig.spec.json
-│   └── typings.d.ts
-├── tsconfig.json
-└── tslint.json
-
-```
-## Browser Support
-
-At present, we officially aim to support the last two versions of the following browsers:
-
-<img src="https://s3.amazonaws.com/creativetim_bucket/github/browser/chrome.png" width="64" height="64"> <img src="https://s3.amazonaws.com/creativetim_bucket/github/browser/firefox.png" width="64" height="64"> <img src="https://s3.amazonaws.com/creativetim_bucket/github/browser/edge.png" width="64" height="64"> <img src="https://s3.amazonaws.com/creativetim_bucket/github/browser/safari.png" width="64" height="64"> <img src="https://s3.amazonaws.com/creativetim_bucket/github/browser/opera.png" width="64" height="64">
-
-
-## Resources
-- [Live Preview](https://demos.creative-tim.com/paper-kit-2-angular/)
-- Download Page: https://www.creative-tim.com/product/paper-kit-2-angular
-- Documentation is [here](https://demos.creative-tim.com/paper-kit-2-angular/documentation/tutorial)
-- License Agreement: https://www.creative-tim.com/license
-- Support: https://www.creative-tim.com/contact-us
-- Issues: [Github Issues Page](https://github.com/creativetimofficial/paper-kit-2-angular/issues)
+![ezgif com-video-to-gif](https://user-images.githubusercontent.com/35809841/78868986-77e2f380-7a61-11ea-9cfd-56bdc75e0654.gif)
 
-## Reporting Issues
-We use GitHub Issues as the official bug tracker for the Paper Kit 2 Angular. Here are some advices for our users that want to report an issue:
+While it's forking, it will look like this.
 
-1. Make sure that you are using the latest version of the Paper Kit 2 Angular. Check the CHANGELOG from your dashboard on our [website](https://www.creative-tim.com/).
-2. Providing us reproducible steps for the issue will shorten the time it takes for it to be fixed.
-3. Some issues may be browser specific, so specifying in what browser you encountered the issue might help.
+![ezgif com-video-to-gif (1)](https://user-images.githubusercontent.com/35809841/78869159-bd9fbc00-7a61-11ea-976c-8aabb1e42099.gif)
 
-## Technical Support or Questions
+### Clone the project you forked
 
-If you have questions or need help integrating the product please [contact us](https://www.creative-tim.com/contact-us) instead of opening an issue.
+The URL of the project you cloned would look like 
+(Replace [your_username] with your github username.)
+``` https://github.com/[your_username]/codeJunction-site```
 
-## Licensing
+You can clone it to your local machine by typing and entering the below command in your terminal. 
+(Replace [your_username] with your github username.)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/gh/codeJunctionTeam/codeJunction-site/pullRequest?prid=5333118)